README.md

```diff
@@ -19,23 +19,38 @@
   <a href="https://app.bors.tech/repositories/28499"><img src="https://bors.tech/images/badge_small.svg" alt="Bors enabled"></a>
 </p>
 
-<p align="center">⚡ The MeiliSearch API client written in Swift 🍎</p>
+<p align="center">⚡ The MeiliSearch API client written for Swift 🍎</p>
 
-**MeiliSearch-Swift** is a client for **MeiliSearch** written in Swift.
+**MeiliSearch Swift** is the MeiliSearch API client for Swift developers.
 
 **MeiliSearch** is an open-source search engine. [Discover what MeiliSearch is!](https://github.com/meilisearch/MeiliSearch)
-For more information about features go to [our Swift documentation](https://meilisearch.github.io/meilisearch-swift/).
 
-## Get started
+## Table of Contents <!-- omit in toc -->
 
-### Cocoapods
+- [📖 Documentation](#-documentation)
+- [🔧 Installation](#-installation)
+- [🎬 Getting Started](#-getting-started)
+- [🤖 Compatibility with MeiliSearch](#-compatibility-with-meilisearch)
+- [💡 Learn More](#-learn-more)
+- [⚙️ Development Workflow and Contributing](#️-development-workflow-and-contributing)
+- [📜 Demos](#-demos)
+
+## 📖 Documentation
+
+For more information about this API see [our Swift documentation](https://meilisearch.github.io/meilisearch-swift/).
+
+For more information about MeiliSearch see our [Documentation](https://docs.meilisearch.com/learn/tutorials/getting_started.html) or our [API References](https://docs.meilisearch.com/reference/api/).
+
+## 🔧 Installation
+
+### With Cocoapods <!-- omit in toc -->
 
 [CocoaPods](https://cocoapods.org/) is a dependency manager for Cocoa projects.
 
 **MeiliSearch-Swift** is available through [CocoaPods](https://cocoapods.org). To install
```

Comment:
It's the exact same link in almost the same sentence may be just one it's enough?